### PR TITLE
Fix logic exception for transaction deposits

### PR DESCRIPTION
**Pull request by d-moreira**
This PR fixes the logic exception for transaction deposits.

### CODE REVIEW

**Comment by SandroMachado** [comment]
LGTM

File: src/app/src/main/java/com/uphold/uphold_android_sdk/model/Transaction.java [hunk 1]
```diff
@@ -73,6 +73,20 @@ public Transaction(String id, String createdAt, Denomination denomination, Desti
     }
 
     /**
+     * Gets the card id path to commit the transaction.
+     *
+     * @return the card id path to commit the transaction.
+     */
+
+    private String getCardIdPath() {
+        if (this.getOrigin().getCardId() == null) {
```
Comment (ruipenso) [inline src/app/src/main/java/com/uphold/uphold_android_sdk/model/Transaction.java]: `"deposit".equalsIgnoreCase(this.getType())`

**Comment by d-moreira** [comment]
Updated @SandroMachado and @ruipenso.

File: src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java [hunk 2]
```diff
@@ -175,6 +275,7 @@ public void commitShouldReturnTheLogicExceptionTransactionCouldNotBeCommited() t
         Assert.assertEquals(exception.getMessage(), "This transaction cannot be committed, because the current status is completed");
     }
 
+
```
Comment (SandroMachado) [inline src/app/src/androidTest/java/com/uphold/uphold_android_sdk/test/integration/model/TransactionTest.java]: Remove this new line.

**Comment by d-moreira** [comment]
Updated @SandroMachado.

**Pull request merged**